app/jobs/search_keyword_job.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class SearchKeywordJob < ApplicationJob
+  queue_as :default
+
+  def perform(keyword_id)
+    keyword = Keyword.find(keyword_id)
+    html = GoogleSearchService.new(keyword: keyword.keyword).call
+    utf_8_html = html.force_encoding('iso8859-1').encode('utf-8')
+    keyword.add_html(utf_8_html)
+  rescue ActiveRecord::RecordNotFound, GoogleSearch::Errors::SearchKeywordError, ActiveRecord::StatementInvalid
+    keyword.update_status(failed)
```

Comment:
I think this would raise an error. We should pass the symbol here. How about adding a test for this case to cover failed case?
```suggestion
    keyword.update_status(:failed)
```

Reply:
Fixed in [`cfb39e5` (#45)](https://github.com/suho/google-search-ruby/pull/45/commits/cfb39e559ebf6e44f9cc53cad7f55df6ab0567fa)